lib/modules/dosomething/dosomething_metatag/dosomething_metatag.module

```diff
@@ -190,6 +220,27 @@ function dosomething_metatag_get_permalink_vars() {
   );
   drupal_add_html_head($element, 'twitter:url');
 
+  // Facebook.
+  $facebook_data = array(
+    'type' => 'dosomething_facebook:campaign',
+    'url' => $share_url,
+    'title' => $node->title,
```

Comment:
@angaither hmm this seems to be pulling the right translation. Maybe because I'm not pulling it from `title_field`? It seems the issue that is referenced is for translatable campaign collections?

![screen shot 2015-12-10 at 2 38 27 pm](https://cloud.githubusercontent.com/assets/1700409/11726277/e7c50530-9f4b-11e5-9ddc-bc710227b4ac.png)
![screen shot 2015-12-10 at 2 38 18 pm](https://cloud.githubusercontent.com/assets/1700409/11726276/e7c1ec06-9f4b-11e5-8d2c-dff05f6df529.png)
![screen shot 2015-12-10 at 2 38 03 pm](https://cloud.githubusercontent.com/assets/1700409/11726275/e7c0bf98-9f4b-11e5-885e-eb5092cc8f39.png)